ocaml/typing/env.mli

```diff
@@ -489,6 +489,14 @@ val imports: unit -> Import_info.t list
 (* may raise Persistent_env.Consistbl.Inconsistency *)
 val import_crcs: source:string -> Import_info.t array -> unit
 
+(* Return the set of imports represented as parameters, along with the
+   local variable representing each *)
```

Comment:
It's not actually out of date, though it should specifically say runtime parameters (that is, the parameters that the compiled functor takes). Spelling out which imports become runtime parameters is a good idea, though, so I've added that parenthetically:

```
(* Return the set of imports represented as runtime parameters (namely,
   parameter imports and parameterised imports), along with the
   local variable representing each *)
```

Reply:
Yeah I guess then we need a paragraph somewhere stressing the difference between parameters and runtime parameters? Because in this same file, there is `val register_parameter`, in which the meaning of "parameter" is different.

Then, I think we can rename this function to `runtime_parameters`, for a better correspondence with `register_parameters`.

Reply:
Essentially I hope that at least for each single `.mli` we have uniform terminilogies.